packages/typespec-vscode/src/typespec-utils.ts

```diff
@@ -53,3 +53,8 @@ export async function TraverseMainTspFileInWorkspace() {
         .map((uri) => normalizeSlashes(uri.fsPath)),
     );
 }
+
+export function GetVscodeUriFromPath(path: string) {
```

Comment:
camelCase


isn't that what the fileService already gives you too?

Reply:
VscodeUri is encoded. fileService provide non-encoded one
e.g. 
fileService:    c:\\dev\\demoProject\\main.tsp
vscodeuri:     file:///c%3A/dev/demoProject/main.tsp

in lsp server cache, it will use vscodeUri as the key,